src/main/java/bio/terra/service/common/gcs/GcsUriUtils.java

```diff
@@ -85,7 +85,11 @@ public static String getGsPathFromComponents(String bucket, String name) {
   }
 
   public static BlobId getBlobForFlight(String bucket, String name, String flightId) {
-    return BlobId.of(bucket, String.format("%s/%s", flightId, name));
+    return parseBlobUri(getPathForFlight(bucket, name, flightId));
```

Comment:
Its essentially the same operation. `getPathForFlight` will create a gsPath, and then `parseBlobUri` will turn it into a `BlobId1`. I did this because we need both the String path and blob for different things (I dislike this and would like to change it) and since it's done in multiple places, the code is reusable like this.